include/dsn/dist/replication/replica_test_utils.h

```diff
@@ -35,8 +35,12 @@ namespace replication {
 class replica;
 class replica_stub;
 
-extern replica *create_test_replica(
-    replica_stub *stub, gpid gpid, const app_info &app, const char *dir, bool restore_if_necessary);
+extern replica *create_test_replica(replica_stub *stub,
```

Comment:
Your title is wrong because virtual is not used for override

Reply:
update to `polymorphism`